api/v1alpha1/externaldns_webhook.go

```diff
@@ -104,6 +120,9 @@ func (r *ExternalDNS) validateHostnameAnnotationPolicy() error {
 }
 
 func (r *ExternalDNS) validateProviderCredentials() error {
+	if ocpVersion == OcpMajorVersion4 {
```

Comment:
We should not skip the validation for Infoblox and BlueCat DNS providers because CCO covers only AWS/Azure/GCP.